lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1875,7 +1875,7 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :default => 10000,
           :public => true,
           :type => Integer,
-          :allowed_from_server => false,
+          :allowed_from_server => true,
```

Comment:
This turns out to be necessary for how we translate back from the `harvest_limits` in the connect response to the configs. You can see the other faster harvest config values (look for `max_samples_stored` ) also default to `:allowed_from_service => true`